tree/dataframe/inc/ROOT/RDF/RCutFlowReport.hxx

```diff
@@ -58,6 +58,7 @@ public:
    const TCutInfo &At(std::string_view cutName) { return operator[](cutName); }
    const_iterator begin() const { return fCutInfos.begin(); }
    const_iterator end() const { return fCutInfos.end(); }
+   void Merge(RCutFlowReport &other);
```

Comment:
`const RCutFlowReport &other`?

Reply:
I didn't put the `const` keyword in the end because the `operator[]` is a non-const function. Indeed putting the `const` keyword produces:
```
/home/vpadulan/Programs/root/lcg96bpy3/root/tree/dataframe/src/RCutFlowReport.cxx: In member function ‘void ROOT::RDF::RCutFlowReport::Merge(const ROOT::RDF::RCutFlowReport&)’:
/home/vpadulan/Programs/root/lcg96bpy3/root/tree/dataframe/src/RCutFlowReport.cxx:69:50: error: passing ‘const ROOT::RDF::RCutFlowReport’ as ‘this’ argument discards qualifiers [-fpermissive]
       auto other_info = other[this_info.GetName()];
```
I could go further and change also the `operator[]` to be a const function, but maybe it's not a smart move. Looking for your opinion on this